code/src/index.css

```diff
@@ -1,13 +1,263 @@
-  -moz-osx-font-smoothing: grayscale;
+  background-color: #5A56BF;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.outerWrapper {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
 }
+
+.headerComponent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.headerComponent h1 {
+  font-family:'Segoe UI', Tahoma, Verdana, sans-serif;
+  color: #5A56BF;
+  font-size: 45px;
+  background-color: #F2E18D;
+  padding: 10px;
+  margin-bottom: 0%;
+}
+
+.headerComponent h2 {
+  color: #F2E18D;
+  font-size: 40px;
+  margin-bottom: 0%;;
+}
+
+.nameComponent {
+  font-family:'Segoe UI', Tahoma, Verdana, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 20px;
+  padding: 15px;
+}
+
+.nameComponent p {
+  padding-bottom: 15px;
+  color: #F2E18D;
+  font-size: 25px;
+}
+
+.nameComponent h6, .goingComponent h6, .rewardComponent h6 {
+  color: #F2E18D;
+  font-size: 15px;
+  font-weight: normal;
+  text-align: center;
+}
+
+.inputField {
+  font-family:'Segoe UI', Tahoma, Verdana, sans-serif;
+  padding: 10px;
+  border-radius: 20px;
+  margin: 10px;
+  border: 3px black solid;
+  font-size: 20px;
+  font-weight: bold;
+} 
+
+.goingComponent {
+  display: flex;
+  flex-direction: column;
+  row-gap: 15px;
+  font-family: 'Segoe UI', Tahoma, Verdana, sans-serif;
+  align-items: stretch;
+  border-radius: 20px;
+  padding: 30px;
+  font-size: 20px;
+  color: #F2E18D;
+  margin-bottom: 0%;
+}
+
+.goingComponent h2 {
+  font-family:'Segoe UI', Tahoma, Verdana, sans-serif;
+  line-height: 120%;
+  margin-top: 0px;
+  color: #5A56BF;
+  font-size: 45px;
+  background-color: #F2E18D;
+  padding: 10px;
+}
+
+.goingComponent input {
+  margin-right: 20px;
+  accent-color: #F2E18D;
+}
+
+.rewardComponent {
+  font-family: 'Segoe UI', Tahoma, Verdana, sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.rewardComponent h2 {
+  font-family:'Segoe UI', Tahoma, Verdana, sans-serif;
+  text-align: center;
+  color: #5A56BF;
+  font-size: 45px;
+  background-color: #F2E18D;
+  padding: 10px;
+}
+
+.selectDropDown {
+  font-size: 20px;
+  padding: 5px;
+  width: 70%;
+  border-radius: 10px;
+  border: 3px black solid;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.resultComponent {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-size: 17px;
+}
+
+.results {
+  font-family:'Segoe UI', Tahoma, Verdana, sans-serif;
+  font-size: 25px;
+  padding: 30px;
+  line-height: 140%;
+  color: #F2E18D;
+  border: 3px black solid;
+  border-radius: 30px;
+  margin-bottom: 30px;
+}
+
+.resultComponent h2 {
+  font-family:'Segoe UI', Tahoma, Verdana, sans-serif;
+  text-align: center;
+  color: #5A56BF;
+  font-size: 45px;
+  background-color: #F2E18D;
+  padding: 10px;
+}
```

Comment:
There is a few h2s here that could've been stacked together aswell.  However, it's important to keep in mind that grouping selectors together can make the code harder to read and maintain, especially if you have a lot of selectors with different styles grouped together!  So I trust your judgement on the balance between conciseness and readability!